openmls/src/group/mls_group/proposals.rs

```diff
@@ -195,6 +217,106 @@ impl StagedProposalQueue {
     }
 }
 
+pub struct StagedAddProposal<'a> {
+    add_proposal: &'a AddProposal,
+    sender: &'a Sender,
+}
+
+impl<'a> StagedAddProposal<'a> {
+    pub(crate) fn try_from_staged_proposal(staged_proposal: &'a StagedProposal) -> Option<Self> {
```

Comment:
Doesn't `try` imply that it will result in an error upon failure? Seems a bit odd that this would return `Option`.